timescaledb/how-to-guides/compression/backfill-historical-data.md

```diff
@@ -1,18 +1,28 @@
 # Backfill historical data on compressed chunks
 
 In the [TimescaleDB extras][timescaledb-extras] GitHub repository, we provide
-explicit functions for [backfilling batch data to compressed
-chunks][timescaledb-extras-backfill], which is useful for inserting a *batch*
-of backfilled data (as opposed to individual row inserts). By "backfill", we
-mean inserting data corresponding to a timestamp well in the past, which given
-its timestamp, already corresponds to a compressed chunk.
+explicit functions for
+[backfilling batch data to compressed chunks][timescaledb-extras-backfill],
+which is useful for inserting a *batch* of backfilled data (as opposed to
+individual row inserts). By "backfill", we mean inserting data corresponding to
+a timestamp well in the past, which given its timestamp, already corresponds to
+a compressed chunk.
 
 In the below example, we backfill data into a temporary table; such temporary
 tables are short-lived and only exist for the duration of the database
 session. Alternatively, if backfill is common, one might use a normal table for
 this instead, which would allow multiple writers to insert into the table at
 the same time before the `decompress_backfill` process.
 
+<procedure>
+
+## Backfilling using the `decompress_backfill` procedure
+
+<highlight type="important">
+The `decompress_backfill` function doesn't support distributed hypertables. To
+backfill a distributed hypertable, manually decompress the chunks.
```

Comment:
By adding "...before the data is inserted." to the end of the sentence, the user might be able to understand it better.